src/browserlib/reffy.js

```diff
@@ -25,6 +27,8 @@ window.reffy = Object.assign(
     extractReferences,
     extractLinks,
     canonicalizeUrl,
-    canonicalizesTo
+    canonicalizesTo,
+    createOutline,
```

Comment:
do we need to expose `createOutline` in the reffy object? as far as I can see, this is only used in the context of the test, which I assume can be avoided.

Reply:
It can be avoided. It is a bit of a pain, because tests run in `about:blank` pages, and there's no way to import a JS module directly in such pages (the browser cannot resolve `import` paths), so one has to convert the JS module to a regular JS function first. Anyway, I now use Rollup to do that conversion at the beginning of the test.